ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -85,61 +85,62 @@ extern uint8_t rasters[][13];
  * [Character (ASCII)][Height][Width]
  */
 static singleLineSprite_t singleLineSprite;
+int64_t start_time, stop_time;
 
 /**
  * @brief There can be different schemes for bounding boxes.
  */
 typedef enum
 {
-  TFLITE_SSD_BOUNDING_BOX = 0,
-  TF_SSD_BOUNDING_BOX = 1,
+  MOBILENET_SSD_BOUNDING_BOX = 0,
+  MOBILENET_SSD_PP_BOUNDING_BOX = 1,
   OV_PERSON_DETECTION_BOUNDING_BOX = 2,
   OV_FACE_DETECTION_BOUNDING_BOX = 3,
   BOUNDING_BOX_UNKNOWN,
 } bounding_box_modes;
 
 /**
- * @brief TF SSD Output tensor feature mapping.
+ * @brief MOBILENET SSD PostProcess Output tensor feature mapping.
  */
 typedef enum
 {
-  TF_SSD_BBOX_IDX_LOCATIONS = 0,
-  TF_SSD_BBOX_IDX_CLASSES = 1,
-  TF_SSD_BBOX_IDX_SCORES = 2,
-  TF_SSD_BBOX_IDX_NUM = 3,
-  TF_SSD_BBOX_IDX_UNKNOWN
-} tf_ssd_bbox_idx_t;
+  MOBILENET_SSD_PP_BBOX_IDX_LOCATIONS = 0,
+  MOBILENET_SSD_PP_BBOX_IDX_CLASSES = 1,
+  MOBILENET_SSD_PP_BBOX_IDX_SCORES = 2,
+  MOBILENET_SSD_PP_BBOX_IDX_NUM = 3,
+  MOBILENET_SSD_PP_BBOX_IDX_UNKNOWN
+} mobilenet_ssd_pp_bbox_idx_t;
 
 /**
  * @brief List of bounding-box decoding schemes in string
  */
 static const char *bb_modes[] = {
-  [TFLITE_SSD_BOUNDING_BOX] = "tflite-ssd",
-  [TF_SSD_BOUNDING_BOX] = "tf-ssd",
+  [MOBILENET_SSD_BOUNDING_BOX] = "mobilenet-ssd",
+  [MOBILENET_SSD_PP_BOUNDING_BOX] = "mobilenet-ssd-postprocess",
```

Comment:
Can you somehow keep the backward compatibility? (it does not need to be documented, it can be a hidden feature for old app codes)

Reply:
The commit for backward compatibility has been added. Please check :)